### DISCUSSION TITLE
Is v1 documentation deployed anywhere?

### DISCUSSION THREAD

**Discussion by robations** [vercel/turborepo#9069] [Help] [answered]
### Summary

I'm trying to find the docs for v1. (Keen to upgrade at some point but I don't need extra work right now...)

### Additional information

I looked on https://turbo.build/repo/docs and searched for v1 but didn't see anything or any version switch

I suppose I can find a way to build it locally, but much easier if this is already in the wild.

I also tried the Wayback Machine, but I'm getting a lot of 500 errors at the moment.

**Comment by romellem** [comment, marked as answer]
I too was annoyed by this, so I deployed my own on Github Pages:

- **https://romellem.github.io/turbo-v1-docs/**

It took a while to build and deploy this, actually. I was seeing the same problem as described here - https://github.com/vercel/next.js/discussions/45443 - but the actual issue was: the `_next/` folder wasn't being included in the default gh-pages build/deploy process. Because it uses [jekyll](https://jekyllrb.com/) under the hood, by convention it was looking for a `_sites/` directory. 

To fix all of this, I changed the gh-pages deployment to a Github Actions based one, and set it up to deploy _all_ of my static assets (I was committing the "export" of the next.js site after it was completed).

Took forever but hopefully I never have to touch this again, and this can help other people looking for the v1 docs.

**Reply by robations** [reply]
Wow, good effort... I managed to upgrade my own project several months back, but looks like there are other people looking for this.
